apps/accounts/automations/check_account/manager.py

```diff
@@ -267,7 +267,7 @@ def get_report_template(self):
         return "accounts/check_account_report.html"
 
     def print_summary(self):
-        tmpl = _("\n---\nSummary: \nok: {}, weak password: {}, leaked password: {}, "
+        tmpl = _("---\nSummary: \nok: {}, weak password: {}, leaked password: {}, "
                  "repeated password: {}, no secret: {}, using time: {}s").format(
             self.summary["ok"],
             self.summary[RiskChoice.weak_password],
```

Comment:
No significant differences were found between the Python versions used to run this code. However, it's important to ensure that any changes made to the script after June 2021 should be compatible with all supported Python versions up until then due to potential API changes or incompatible behavior. Always test before making these modifications in production applications.